src/app/core/http-services/get-user-by-login.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
```

Comment:
On click, the button is properly greyed out but there's no loading indicator, have you tried providing the input `[loading]` as specified in [primeng documentation](https://www.primefaces.org/primeng/showcase/#/button) ?

Reply:
This version of primeng doesn't have loading feature for button. I've got task from Damien for update primeng but in specific branch